ui/.eslintrc

```diff
@@ -1,8 +1,17 @@
 {
   "root": true,
-  "parser": "babel-eslint",
+  "parser": "@babel/eslint-parser",
```

Comment:
It's better to use `@typescript-eslint/parser` instead. It's faster and has native TS support